xml/ay_software.xml

```diff
@@ -81,23 +81,23 @@
       <term>SUSE-manager-server</term>
       <listitem>
        <para>
-        SUSE Manager Server
```

Comment:
How the AY folks want to handle this for older SuMa products vs. the new SMLM product is still under discussion, see https://bugzilla.suse.com/show_bug.cgi?id=1239854 

Therefore I only updated the clear text names for the products (and replaced them with the new entities) and did not modify the available values for the productname tag yet...